config/samples/banzaicloud_v1beta1_kafkacluster.yaml

```
@@ -5,6 +5,8 @@ metadata:
     controller-tools.k8s.io: "1.0"
   name: kafka
 spec:
+  # specify if the cluster is in KRaft mode or not, default is true
```

```suggestion
  # specify if the cluster is in KRaft mode or not, default is false
```

But this file might be generated.